recirq/qaoa/placement.py

```diff
@@ -57,36 +33,6 @@ def calibration_data_to_graph(calib_dict: cg.Calibration) -> nx.Graph:
     return err_graph
 
 
-def _qubit_index_edges(device: cirq.Device):
```

Comment:
We probably want to keep this, but mark it deprecated once cirq releases catch up.

Reply:
ok!